src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,52 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_CAPACITY = 10;
+    private Object[] keyArray = new Object[MAX_CAPACITY];
+    private Object[] valueArray = new Object[MAX_CAPACITY];
+    private int size = 0;
+
     @Override
     public void put(K key, V value) {
+        int count = counter(key);
+        if (count == -1) {
+            keyArray[size] = key;
+            valueArray[size] = value;
+            size++;
+        } else if (count >= 0 && count <= size - 1) {
+            valueArray[count] = value;
+        }
     }
 
     @Override
     public V get(K key) {
-        return null;
+        int count = counter(key);
+        if (count == -1) {
+            return (V) null;
+        }
+        return (V) valueArray[count];
     }
 
     @Override
     public int size() {
-        return -1;
+        return size;
+    }
+
+    public int counter(K key) {
```

Comment:
1. Method names should contain verbs for better code readability
2. Please keep public only official class logic. Details of realization shouldn't be public
```suggestion
    private int getIndex(K key) {
```